### ISSUE TITLE
Need to get the underlyingQueue. Make OperationQueue public?

### ISSUE THREAD

**Issue by BabyAzerty** [FabrizioBrancati/Queuer#6]

**Comment by FabrizioBrancati** [comment]
Just to know, why would you like to access OperationQueue?

**Comment by BabyAzerty** [comment]
I have a BackgroundHelper which fetches remote data using Alamofire in such way:

```
let operation = ConcurrentOperation {
            let semaphore = Semaphore()
            DispatchQueue.main.async {  // This is needed because Alamofire uses DispatchQueue.main and Realm must be init on the same thread
                let realmManager = RealmManager.persistentRealm()
                manager = UsersManager(realmManager: realmManager)
                manager?.apiGetAccount { _ in
                    defer { semaphore.continue() }
                }
            }
            semaphore.wait()
        }
```

The thing is that Alamofire's completion block runs on DispatchQueue.main unless I give it a custom thread.

Normally, if I don't mistaken, I can get the current thread of an OperationQueue using "underlyingQueue" variable.

Maybe you have another way of doing so?

**Comment by FabrizioBrancati** [comment]
I have pushed your desired change in the `develop` branch.
Remeber to set the `underlyingQueue` to `DispatchQueue.main` before adding it to the OperationQueue, as wrote [here](https://developer.apple.com/documentation/foundation/operationqueue/1415344-underlyingqueue) 😉 

Test it and when you will say that everything is ok I will push a new version 👍

**Comment by FabrizioBrancati** [comment]
I have released [Queuer 1.3.1](https://github.com/FabrizioBrancati/Queuer/releases/tag/1.3.1) with your fix inside!
Let me know if is everything ok

**Comment by BabyAzerty** [comment]
Thanks, I will try it out this week and let you know if there is a problem or not 👍
